pandas/core/dtypes/common.py

```diff
@@ -655,24 +655,38 @@ def is_dtype_equal(source, target) -> bool:
 
     Parameters
     ----------
-    source : The first dtype to compare
-    target : The second dtype to compare
+    source : dtype
+        The first dtype to compare
+    target : dtype
+        The second dtype to compare
 
     Returns
     -------
     boolean
         Whether or not the two dtypes are equal.
 
+    See Also
+    --------
+    pandas.api.types.is_categorical_dtype : Check whether the provided array or dtype
```

Comment:
```suggestion
    api.types.is_categorical_dtype : Check whether the provided array or dtype
```

Also for the others.